rascaline/src/calculators/lode/spherical_expansion.rs

```diff
@@ -729,4 +814,33 @@ mod tests {
             max_relative=1e-4
         );
     }
+
+    #[test]
+    fn central_atom_contribution() {
+        let potential_exponents = [0, 1, 2, 6];
+
+        // Reference values taken from pyLODE
+        let reference_vals = [
+            [7.09990773e-01, 6.13767550e-01, 3.34161655e-01, 8.35301652e-02, 1.78439072e-02, -3.44944648e-05],
+            [1.69193719, 2.02389574, 2.85086136, 3.84013091, 1.62869125, 7.03338899],
+            [1.00532822, 1.10024472, 1.34843326, 1.19816598, 0.69150744, 1.2765415],
+            [0.03811939, 0.03741200, 0.03115835, 0.01364843, 0.00534184, 0.00205973]];
+
+        for (i, p) in potential_exponents.iter().enumerate(){
+
+            let mut spherical_expansion = LodeSphericalExpansion::new(LodeSphericalExpansionParameters {
+                cutoff: 5.0,
+                k_cutoff: None,
+                max_radial: 6,
+                max_angular: 2,
+                atomic_gaussian_width: 1.0,
+                center_atom_weight: 1.0,
+                radial_basis: LodeRadialBasis::splined_gto(1e-8),
+                potential_exponent: *p,
```

Comment:
FYI, you can remove the need for a dereference here (i.e. use `potential_exponent: p`) by changing the pattern in the for loop to `for (i, &p) in potential_exponents.iter().enumerate() {`. But this is fine as well!